infra/helper.py

```diff
@@ -417,9 +417,9 @@ def build_image(args):
 
 
 def build_fuzzers_impl(project_name, clean, engine, sanitizer, architecture,
-                       env_to_add, source_path):
+                       env_to_add, source_path, no_cache=True):
```

Comment:
Isn't defaulting to `True` changing the way `build_fuzzers` works? Please change it to `False`